contracts/TokenHub.sol

```diff
@@ -857,10 +329,6 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
     }
 
     function withdrawStakingBNB(uint256 amount) external override returns (bool) {
```

Comment:
remove this in `ITokenHub` file

Reply:
it will cause compile error since `TokenHub is ITokenHub`